docs/pages/eas-update/rollouts.mdx

```diff
@@ -2,24 +2,29 @@
 
-Rollouts allow deploying a new branch to a specific channel incrementally. With rollouts, we can roll out one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to your production environment.
+To start a rollout, we use `npx eas-cli@latest` or ensure we have version \<TODO\> or above for `eas-cli`. Then we run the following command:
 
-To initiate a rollout, use the following command:
+<Terminal cmd={['npx eas-cli@latest channel:rollout [channel]']} />
 
-<Terminal cmd={["eas channel:rollout [channel]"]} />
+An interactive guide will assist us in selecting a channel, choosing a branch for the rollout, and setting the percentage of users for the rollout. To increase or decrease the rollout amount, we run the command again and choose the `Edit` option to adjust the rollout percentage.
 
-The interactive prompts will guide you through choosing a channel, a branch to begin rolling out, and the percentage of your users to rollout to. To increase or decrease the rollout amount, run the `eas channel:rollout [channel]` command again to adjust the rollout percentage.
+### Ending a rollout
 
-Once you feel confident with your new branch, or if you want to revert to your original branch, you can end the rollout with the `--end` flag:
+Two methods are available to end a rollout when we choose the `End` option in the interactive guide:
 
-<Terminal cmd={["eas channel:rollout [channel] --end"]} />
+- Republish and revert: This option can be used once we are confident with the update served from our new branch. By choosing this, we'll republish the update onto the old branch and revert users back to it.
+- Revert: Disregard the updates on the new branch and return users to the old branch.
 
-Facts that will help you when working with rollouts:
-- You can only have one branch rolling out on a channel at a single time.
-- To see the state of the rollout, use the `eas channel:view [channel]` command.
-- When a rollout is in progress, you can publish updates to both branches by running `eas update --branch [branch]`, for example.
-- You cannot use `eas update --channel [channel]` when a rollout is in progress, since it cannot know which branch to associate the update with.
+### Working with rollouts
 
+- Only one branch can be rolled out on a channel at a single time.
+- To see the state of the rollout, we use the `eas channel:rollout [channel]` command.
+- When a rollout is in progress, we can publish updates to both branches by running `eas update --branch [branch]`, for example.
+- `eas update --channel [channel]` cannot be used when a rollout is in progress, since it cannot know which branch to associate the update with.
```

Comment:
```suggestion
- Only one branch can be rolled out on a channel at a single time.
- To see the state of the rollout, use the `eas channel:rollout [channel]` command.
- When a rollout is in progress, you can publish updates to both branches by running `eas update --branch [branch]`, for example.
- `eas update --channel [channel]` cannot be used when a rollout is in progress since it cannot know which branch to associate the update with.
```

I'm confused by this phrase: "... to both branches...". Is it meant to imply here both the rollout branch and the current branch? If yes, I'd change this to "... to both rollout and current branches...".

Reply:
i changed this to "... to both rollout and current branches...".